### PR TITLE
Convert amount in transfer - Closes #482

### LINKED ISSUE #482: Amount should be converted before creating transactions using lisk-elements
**Expected behavior:** Amounts provided in whole LSK for transfer transactions should be converted when passed to lisk-elements.

**Actual behavior:** They're passed as-is.

### Versions

1.0.0

### PR DESCRIPTION
**Pull request by willclarktech**
### What was the problem?

We were passing lisk-js the amount in whole LSK instead of the smallest denomination.

### How did I fix it?

I added a `normalizeAmount` helper function. Ideally we would have this function in lisk-elements using bignum, but we can do that post-1.0.0.

### How to test it?

`create transaction transfer 100 123L`

### Review checklist

* The PR solves #482 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/specs/utils/helpers.js [hunk 1]
```diff
@@ -268,6 +268,72 @@ describe('utils helpers', () => {
...
+					() => {
+						Then(
+							'it should return the normalized amount',
+							then.itShouldReturnTheNormalizedAmount,
+						);
+					},
+				);
+			},
+		);
+		Given(
```
Comment (shuse2) [inline test/specs/utils/helpers.js]: It might be good to add edge cases where we have decimal places more than 8

File: src/utils/helpers.js [hunk 2]
```diff
@@ -80,6 +81,16 @@ export const validateAmount = amount => {
 	return true;
 };
 
+export const normalizeAmount = amount => {
```
Comment (shuse2) [inline src/utils/helpers.js]: this function seems pretty cool =)

File: test/specs/utils/helpers.js (hunk 1 above)
Comment (willclarktech) [reply test/specs/utils/helpers.js]: I thought about calling `validateAmount` in here, but that would seem to change the locus of responsibility: right now it's in the action, along with validating the address.

Comment (shuse2) [reply test/specs/utils/helpers.js]: not necessarily about `validAmount`, but as for `unit test` I think it's good to expect error thrown for when length of `postArray` is too long.
```
const pad = new Array(DECIMAL_PLACES - postArray.length).fill('0');
```
